src/Microsoft.VisualStudio.ProjectSystem.Managed/Microsoft.VisualStudio.ProjectSystem.Managed.csproj

```diff
@@ -88,6 +88,9 @@
     <Compile Include="ProjectSystem\Rules\AnalyzerReference.xaml.cs">
       <DependentUpon>AnalyzerReference.xaml</DependentUpon>
     </Compile>
+    <Compile Include="ProjectSystem\Rules\ProjectDebugger.xaml.cs">
```

Comment:
For the setup authoring, it doesn't look like we deploy the .cs files at all. They're just there for IDE features, like the suppressions and turning off code coverage.